packages/server/graphql/public/types/CreateGcalEventInput.ts

```diff
@@ -25,13 +25,14 @@ const CreateGcalEventInput = new GraphQLInputObjectType({
       type: new GraphQLNonNull(GraphQLInt),
       description: 'The end dateTime of the meeting'
     },
-    inviteTeam: {
-      type: new GraphQLNonNull(GraphQLBoolean),
-      description: 'Whether the users in the team should be invited in gcal'
-    },
     timeZone: {
       type: new GraphQLNonNull(GraphQLString),
       description: 'The timeZone of the meeting'
+    },
+    invitees: {
+      type: new GraphQLList(GraphQLString),
```

Comment:
Should we do it like this: `new GraphQLList(new GraphQLNonNull(GraphQLEmailType)))` ?

Reply:
Sounds good, updated 👍